package.json

```
@@ -37,13 +37,21 @@
     "ember-modifier": "^3.0.0 || ^4.0.0",
     "intersection-observer-admin": "^0.3.4"
   },
+  "peerDependencies": {
+    "@ember/test-helpers": "^3.0.0 || ^4.0.0"
```

```suggestion
    "@ember/test-helpers": ">= 3.0.0"
```
To account for future majors, if you want